### ISSUE TITLE
Interface 'Illuminate\Contracts\Validation\Rule' not found (Laravel 5.4)

### ISSUE THREAD

**Issue by mirko77** [langleyfoxall/laravel-nist-password-rules#28]
```
FatalErrorException in SequentialCharacters.php line 13:
Interface 'Illuminate\Contracts\Validation\Rule' not found

```

when calling

```

// Register
        $this->validate($request, [
            'email' => 'required',
            'password' => PasswordRules::register($request->email),
        ]);

```

on Laravel 5.4

Any suggestions?

**Comment by DivineOmega** [comment]
It looks like the `Rule` interface we're using here is only available on Laravel 5.5 and above. We may need to change the package's minimum Laravel version if there is no alternative interface.

**Comment by DivineOmega** [comment]
I'm afraid we've had to set the minimum supported Laravel version to 5.5 now for ease of maintenance.
